cryptic-game/frontend
# Added control center page for building devices

**Description**
The new page allows players to build computers by selecting hardware components.

**Issue**
Part of #77

**Testing Instructions**
- Buy hardware items in the Hardware Shop app
- Open up the "Computers" sidebar menu
- Click on "Build new device"
- Configure the new computer by selecting the preferred parts
- Click "Check Compatibility"
- Click "Build"
